### ISSUE TITLE
daemon-reload not required

### ISSUE THREAD

**Issue by debian-user-france1** [safing/docs#76]
**What is the bug / inaccuracy?**:
Installing on Linux => Arch Linux

The doc:
```
# Clone the repository
git clone https://github.com/safing/portmaster-packaging

# Enter the repo and build/install the package (it's under linux/)
cd portmaster-packaging/linux
makepkg -is

# Start the Portmaster and enable autostart
sudo systemctl daemon-reload
sudo systemctl enable --now portmaster
```

**The systemctl daemon-reload is not required.**
**What would you expect instead?**:
The doc should be like that:
```
...
# Start the Portmaster and enable autostart
sudo systemctl enable --now portmaster
```

**Comment by p2anav** [comment]
Thanks for reporting. Just curious to learn - why is `systemctl daemon-reload` not required?

**Comment by arch-user-france1** [comment]
I think it is only required if a service file is edited but not when it's created. I might be wrong tho...

**Comment by arch-user-france1** [comment]
And I of course made sure that I tried it without running `systemctl daemon-reload` and it worked so it should be safe-to-remove.

**Comment by dhaavi** [comment]
It seems that `daemon-reload` is only needed in some cases: https://man.archlinux.org/man/systemctl.1#Unit_File_Commands

**Comment by arch-user-france1** [comment]
> It seems that `daemon-reload` is only needed in some cases: https://man.archlinux.org/man/systemctl.1#Unit_File_Commands

Yes because it reloads configurations in case of edits but it makes no sence to execute if there's a new configuration file. It's like purging a cache I guess. So it is definitely not required. Should be removed out of the manual, it's not serious to release a _bloated_ manual.

**Comment by ppacher** [comment]
Hi, 

Thats totally true, daemon-reload is not required for first-time installs but in case a user does an update it **needs** to be done.  Updates of the package are required from time to time as the unit file ist not managed by portmaster and thus does not update automatically. In order to avoid a dedicated section on "how to update the didtro package" daemon-reload is part of the common install section. Also, it should not do any harm (users that mangle with their unit files are expected to already know what they are doing). So IMHO this should be kept as it is.

I'm on my phone (with a broken display) right now so please ignore any typing errors^^

**Comment by arch-user-france1** [comment]
> Hi,
> 
> Thats totally true, daemon-reload is not required for first-time installs but in case a user does an update it **needs** to be done. Updates of the package are required from time to time as the unit file ist not managed by portmaster and thus does not update automatically. In order to avoid a dedicated section on "how to update the didtro package" daemon-reload is part of the common install section. Also, it should not do any harm (users that mangle with their unit files are expected to already know what they are doing). So IMHO this should be kept as it is.
> 
> I'm on my phone (with a broken display) right now so please ignore any typing errors^^

Is a daemon-reload required even after a restart of the device?

**Comment by ppacher** [comment]
Hi, no a restart of the device makes daemon-reload a no-op because systemd does scan the service files on start anyway.
